charts/launch-agent/values.yaml

```diff
@@ -0,0 +1,15 @@
+agent:
+  # W&B API key.
+  apiKey: ""
+  # Container image to use for the agent.
+  image: wandb/launch-agent-dev:latest
+  # Image pull policy for agent image.
+  imagePullPolicy: Always
+  # Resources block for the agent spec.
+  resources:
+    limits:
+      cpu: 1000m
+      memory: 1Gi
```

Comment:
We should probably tell users to override this in our README as it's won't make sense for the majority of workloads.